### DISCUSSION TITLE
Syntax Error when saving active model that sets nothing

### DISCUSSION THREAD

**Discussion by lihe07** [SeaQL/sea-orm#1371] [Q&A] [answered]
I have a ActiveModel like this;
```rust
ActiveModel {
    id; Unchanged(1),
    name: NotSet,
    password: NotSet
}
```
The SQL query for saving this looks like that:
```sql
UPDATE
  `users`
SET
WHERE
  `users`.`id` = ?
```
And that leads to a syntax error (empty set section).

I'm considering whether it's sea-orm's task to determine if the ActiveModel does noting or not.

**Comment by billy1624** [comment, marked as answer]
Hey @lihe07, I think this is something that we need to "fix". For anyone who is reading this please track the progress and discussion at https://github.com/SeaQL/sea-orm/issues/1376
